src/utils/vscode.ts

```diff
@@ -12,6 +12,7 @@ import {
     TextEditor,
     commands,
 } from "vscode";
+import type * as lsp from "vscode-languageserver-types";
```

Comment:
Adding a package just for a Range seems unnecessary. It would be better to directly define LspRange like this:

```typescript
{
    start: { line: number; character: number };
    end: { line: number; character: number };
}
```

Reply:
Yeah I waffled on this, see above :D